src/main/java/com/minecolonies/coremod/colony/requestsystem/requests/StandardRequests.java

```diff
@@ -186,7 +186,16 @@ public ITextComponent getShortDisplayString()
         {
             final IFormattableTextComponent combined = new NonSiblingFormattingTextComponent();
             combined.append(new StringTextComponent(getRequest().getCount() + " "));
-            combined.append(new StringTextComponent("#" + getRequest().getTag().toString()));
+            // getRequest().getTag() is a long string that can't be easily be read by players or turned into a translation key.
+            // Instead, preferentially select the first item from the list.
+            if(!stacks.isEmpty())
```

Comment:
I'm wondering about this, I'd like the input of @OrionDevelopment on this, but all our tag requests are very specific. We could have something like com.minecolonies.coremod.tag.lastPartOfTagKey in the lang file and then use that, so we got it nicely translated

Reply:
That's definitely doable for internal use, and there are only a few item tags used at all right now.  Do you have any concerns about other mods or user/modpack-maker defined recipes or tags?

With the other changes in this commit, the getShortDisplayName version only shows up on the StandardInteraction message.  AbstractWindowRequestTree cycles through the items that will fulfill the request.  Do you prefer that, or would you rather show this new tag translated text?

Reply:
We could always do a check, since this is on the client side, we can check if we have a propper translation, if so we use it, else we use the other.